File: Basic Coding Problems/01 Fundamentals/012 Write a program to enter any number and calculate its square root/C/square_root.c
#include <stdio.h>
#include <math.h>

int main() {
    int number;
    float square_root;

    printf("Enter a number: ");
    scanf("%d", &number);

    square_root = sqrt(number);

    printf("The square root of %d is %f\n", number, square_root);

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/012 Write a program to enter any number and calculate its square root/Java/square_root.java
import java.util.Scanner;

public class square_root {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int number = scanner.nextInt();

        float squareRoot = (float) Math.sqrt(number);

        System.out.println("The square root of " + number + " is " + squareRoot);
    }
}